src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +97,21 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate localDate = LocalDate.now();
+        int date = localDate.compareTo(someDate);
+        return date < 0 ? someDate + " is after " + localDate
```

Comment:
You could use methods isBefore and isAfter with LocalDate. Not sure how critical; this is, but I thought this task is to study those methods...